dymos/transcriptions/pseudospectral/pseudospectral_base.py

```diff
@@ -436,6 +436,54 @@ def configure_defects(self, phase):
                               f'continuity_comp.control_rates:{name}_rate2',
                               src_indices=src_idxs, flat_src_indices=True)
 
+    def setup_duration_balance(self, phase):
+        """
+        Setup the implicit computation of the phase duration.
+
+        Parameters
+        ----------
+        phase : dymos.Phase
+            The phase object to which this transcription instance applies.
+        """
+
+        if phase.time_options['t_duration_balance_options']:
+            self.implicit_duration = True
```

Comment:
Minor nitpick...use `self._implicit_duration` so that its clear that this isn't an attribute that people should be interacting with.